.github/workflows/create_release.yml

```diff
@@ -23,7 +23,7 @@ jobs:
       - name: Set up Go
         uses: actions/setup-go@v3.5.0
         with:
-          go-version: 1.19.5
+          go-version: 1.20.5
```

Comment:
https://go.dev/dl/ suggests 1.20.7 is the current tip.  Does the patch version matter here?  If not, maybe use 0 and explain in the commit message?  If it does, maybe use 1.20.7?

Reply:
This was the version I was using locally.  I'll re-do this with latest.